client/galaxy/style/less/ui.less

```diff
@@ -152,7 +152,7 @@
             font-size: 0.8em;
             font-weight: normal;
             top: -1px;
-            word-break:break-word;
```

Comment:
I am revisiting this for purposes of https://github.com/galaxyproject/galaxy/issues/2591

Should we have a `break-word` on button titles?